test/User-test.js

```diff
@@ -1,31 +1,31 @@
 import { expect } from 'chai';
-import User from '../src/classes/User.js';
-import Recipe from '../src/classes/Recipe.js';
-import { recipeData } from '../src/data/recipes.js';
-import { users } from '../src/data/users.js';
+import User from '../src/classes/User';
+import Recipe from '../src/classes/Recipe';
+import { usersData } from '../src/data/users-sample';
+import { recipeData } from '../src/data/recipes-sample';
+import { ingredientsData } from '../src/data/ingredients-sample';
 
 describe('User', function() {
-  let user;
-  let recipeOne;
-  let recipeTwo;
-  let recipeThree;
-  let tagOne;
-  let tagTwo;
+  let user, recipeOne, recipeTwo, recipeThree, tagOne, tagTwo, tagThree, ingredientsOne, ingredientsTwo, ingredientsThree, nameOne;
```

Comment:
You could take this as a preference, but I wouldn't advise updating to this type of syntax. I personally find it more confusing and harder to read, plus, even though it's a "one-liner", it's a _very long_ one-liner. I've also never seen anyone use this on the job, for what it's worth.